### PR TITLE
Isochrone feature

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: components/Isochrone/IsochroneMapControl.vue
Comment (wazolab) [inline components/Isochrone/IsochroneMapControl.vue:1]: This file isn't used yet. It's purpose is only if we want to add a global Map Control (maybe for later use)

File: composables/useIsochrone.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          id: `${sourceName}-line`,
+          type: 'line',
+          source: sourceName,
+          layout: {
+            'line-join': 'round',
+            'line-cap': 'round',
+          },
+          paint: {
+            'line-color': '#ffffff',
+            'line-width': 3,
```
Comment (frodrigo) [inline composables/useIsochrone.ts:96]: Same color as fill, but with no opacity.

**Comment by wazolab** [comment]
## Feedback
- [x] border same color as fill
- [x] text-halo-width: 2 (test with same color as border)
- [x] Get lighter color
- [x] Try with one layer and use conditional expression
- [x] Icon + text on profile button
- [x] Maybe remove Calculate button (API call on profile selection)
- [x] Improve profile buttons design
- [x] Center map on bbox
- [x] Display isochrone polygon under line / text
- [ ] - [ ] Ask Stéphane about removing Isochrone polygons when changing feature selection
- [ ] Add text explanations in Overlay (see with Stéphane & Vincent)

File: composables/useIsochrone.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    const { coordinates } = feature.geometry as GeoJSON.Point
+    const uniqueID = `isochrone-${profile}-${feature.properties?.metadata.id}`
+
+    const { data, error } = await useAsyncData<ORSData>(
+      uniqueID,
+      () => $fetch(`https://api.openrouteservice.org/v2/isochrones/${profile}`, {
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json, application/geo+json, application/gpx+xml, img/png; charset=utf-8',
+          'Authorization': '5b3ce3597851110001cf6248e176e3d24c3d4a71bff8701252700040',
```
Comment (frodrigo) [inline composables/useIsochrone.ts:176]: As config param.

File: locales/es-ES.ts [hunk 3]
```diff
@@ -188,5 +188,17 @@ export default defineI18nLocale(() => {
       details: 'Más información',
       decline: 'Descenso',
     },
+    isochrone: {
+      trigger: 'Tiempo de viaje',
```
Comment (frodrigo) [inline locales/es-ES.ts:192]:
```suggestion
      trigger: 'Duración de viaje',
```

File: locales/es-ES.ts [hunk 4]
```diff
@@ -188,5 +188,17 @@ export default defineI18nLocale(() => {
...
+      overlay: {
+        title: '¿A dónde puedo ir?',
+        text: 'Visualiza en el mapa las zonas accesibles desde este punto de partida en función de la duración de la modalidad de viaje.',
```
Comment (frodrigo) [inline locales/es-ES.ts:195]:
```suggestion
        text: 'Visualiza en el mapa las zonas accesibles desde este punto de inicio en función de la duración y de la modalidad de viaje.',
```

File: locales/fr-FR.ts [hunk 5]
```diff
@@ -189,5 +189,17 @@ export default defineI18nLocale(() => {
       details: 'En savoir plus',
       decline: 'Refuser',
     },
+    isochrone: {
+      trigger: 'Temps de trajet',
+      overlay: {
+        title: 'Où puis-je aller ?',
+        text: 'Visualiser sur la carte les zones accessibles depuis ce point de départ en fonction de la durée du mode de déplacement.',
```
Comment (frodrigo) [inline locales/fr-FR.ts:196]:
```suggestion
        text: 'Visualiser sur la carte les zones accessibles depuis ce point de départ en fonction de la durée et du mode de déplacement.',
```

File: locales/en-GB.ts [hunk 6]
```diff
@@ -188,5 +188,17 @@ export default defineI18nLocale(() => {
       details: 'More information',
       decline: 'Decline',
     },
+    isochrone: {
+      trigger: 'Travel time',
+      overlay: {
+        title: 'Where can I go ?',
+        text: 'View on map, accessible areas from this point, depending on the duration of travel mode.',
```
Comment (frodrigo) [inline locales/en-GB.ts:195]:
```suggestion
        text: 'View on map, accessible areas from this point, depending on the duration and the travel mode.',
```

**Pull request merged**